nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -498,8 +499,17 @@ def transcribe(
             A list of transcriptions (or raw log probabilities if logprobs is True) in the same order 
             as paths2audio_files
         """
-        if timestamps:
-            raise NotImplementedError("Computing timestamps are not supported for this model yet.")
+        if timestamps is not None and timestamps:
```

Comment:
```suggestion
        if timestamps is not None:
```
otherwise you're missing `timestamps=False` or `timestamps=0` case